### ISSUE TITLE
Phone turns off when viewing licensed media-apps

### ISSUE THREAD

**Issue by GewoonIraj** [Genymobile/scrcpy#2457]
Only with Scrcpy running, when I press play to watch a movie/show on either Netflix or Disney+, scrcpy instantly closes and my phone fully turns off.
It doesn't turn it back on either so I gotta do that manually.

I assume this has something to do with the licensed media these apps provide.

I haven't tried it with other paid-media apps, but I know it does not happen with YouTube.

As I am not entirely just if this is just Android itself being 💀 with this while running Scrcpy, or if it's just Scrcpy itself.

Information if needed:
- PC: Windows 10 Pro (64-bit)
- Phone: Samsung Galaxy S21

**Comment by rom1v** [comment]
> scrcpy instantly closes and my phone fully turns off.

It should just show a black screen, not crash.

Please post the console output.

My guess is that MediaCodec (or other) crashes for some reason on your device when you start Netflix or Disney+.

**Comment by GewoonIraj** [comment]
> > scrcpy instantly closes and my phone fully turns off.
> 
> It should just show a black screen, not crash.
> 
> Please post the console output.
> 
> My guess is that MediaCodec (or other) crashes for some reason on your device when you start Netflix or Disney+.

Well, I can't show the console output as it instantly closes because the phone gets turned off... 😅

When the phone gets disconnected (which happens when it gets turned off), Scrcpy closes along with it.

I'll try to look into this MediaCodec thingy.

**Comment by rom1v** [comment]
> Well, I can't show the console output as it instantly closes because the phone gets turned off

Run `scrcpy-console.bat`.

**Comment by MrShuzo** [comment]
I have the same issue and the console just says `WARN: Device disconnected`. I use it with an Samsung SM-G991B running Android 11 and version 1.17 of scrcpy.

**Comment by rom1v** [comment]
> WARN: Device disconnected


This happens when the connection is lost with the adb server (for whatever reason).

This disconnection affects everything connected to adb. For example, if you run `adb shell` separately, it will terminate at the same time.

**Comment by MrShuzo** [comment]
Yeah, but this happens because my phone just crashes the moment I start playing Netflix or others.
